README.md

```diff
@@ -1,20 +1,47 @@
 # elrrdatasync
 ELRR services which aid in the streaming (Kafka streams) of data from the staging database to the ELRR database.
 
-# Requirements
-For building and running the elrrdatasync you need:
-- JDK 1.8
+# Dependencies
+- Java JDK 17
+- git
 - Maven 3
+- Docker Desktop
+
+# Tools
+- DBeaver 
+- Eclipse or other IDE
+
+# Create Docker Containers
+- Start Docker Desktop
+- docker compose up
+- Check for new containers in Docker Desktop
+   
+# Create and populate PostgreSQL staging schema
+- Start Docker Desktop
+- Open DBeaver
```

Comment:
i already made two notes about this. DBeaver is not a requisite recommendation. Just say SQL Client or terminal.